starcheck/calc_ccd_temps.py

```diff
@@ -169,16 +173,18 @@ def get_ccd_temps(oflsdir, outdir='out',
 
     make_check_plots(outdir, states, times,
                      ccd_temp, tstart, tstop, char=char)
-
     intervals = get_obs_intervals(sc_obsids)
-    obstemps = get_interval_temps(intervals, times, ccd_temp)
+    obsreqs = None
+    if orlist is not None:
+        obsreqs = dict([(o['obsid'], o) for o in read_or_list(orlist)])
```

Comment:
Well it's just that `o` is especially unusual as a variable name because of conflict with `0`.  I would not have complained about `x`.